pr2eus/robot-interface.l

```diff
@@ -352,7 +357,7 @@
          (setq tms (append tms (make-list (- (length avs) (length tms)) :initial-element (car (last tms))))))
      (prog1 ;; angle-vector-sequence returns avs
          avs
-       (while avs
+       (while (cadr avs)
```

Comment:
what do you wan to check ? does avs contain null list, such as `(list #f(....) nil #f(...))` ...?